src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,11 @@
+
+import React from 'react';
+const Todo = props => {
+  return (
+      <div>
```

Comment:
If this component is not doing anything it can be safely removed for the time beign.